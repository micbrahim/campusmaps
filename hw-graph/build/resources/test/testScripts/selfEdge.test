# tests creating an edge starting from a node that points to itself (the same node)

# create graph
CreateGraph g1

# add node
AddNode g1 n1

# create and edge from node to itself
AddEdge g1 n1 n1 e1

# lists child node
ListChildren g1 n1