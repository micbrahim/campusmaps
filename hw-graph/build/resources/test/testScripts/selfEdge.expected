# tests creating an edge starting from a node that points to itself (the same node)

# create graph
created graph g1

# add node
added node n1 to g1

# create and edge from node to itself
added edge e1 from n1 to n1 in g1

# lists child node
the children of n1 in g1 are: n1(e1)